src/service-providers/dataset-level/dataset-level.interface.ts

```diff
@@ -0,0 +1,122 @@
+import { TCondition } from 'service-providers/filtering-regime/filtering-regime.interface'
+import { IBaseDatasetDescriptor } from 'service-providers/vault-level/vault-level.interface'
+
+// dsinfo
+
+export interface IDsInfoArguments {
+  ds: string
+  note?: string
+}
+export interface IDsInfoClass {
+  title: string
+  values: string[]
+}
+
+export interface IDsInfo extends IBaseDatasetDescriptor {
+  meta: object
+  classes: IDsInfoClass[]
+  'unit-groups': string[]
+  cohorts: string[]
+  'export-max-count': number
+}
+
+// ds_list
+
+export interface IDsListArguments {
+  ds: string
+  filter?: string
+  conditions?: TCondition[]
+  dtree?: string
+  code?: string
+  no?: string
+  smpcnt?: string
+}
+export interface IDsList {
+  task_id: string
+}
+
+// reccnt
+
+export interface IReccntArguments {
+  ds: string
+  rec: number
+  details?: number
+  samples?: number[]
+}
+
+export interface ICommonAspectDescriptor {
+  name: string
+  title: string
+  kind: 'norm' | 'tech'
```

Comment:
May be we can use enums instead of string unions

Reply:
Ok, I will change